### PR TITLE
Add specfile

### PR DESCRIPTION
**Pull request by Allda**
This specfile creates two rpms for both python version (2 and 3)

### CODE REVIEW

File: python-lightblue.spec [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Name:           python-lightblue
+Version:        0.1.3
+Release:        1%{?dist}
+Summary:        A Python API for Lightblue database.
+
+Group:          Development/Libraries
+License:        GPLv3
+URL:            https://github.com/Allda/python-lightblue
+Source0:        %{name}-%{version}.tar.gz
```
Comment (sochotnicky) [inline python-lightblue.spec]: ULR to pypi here

File: python-lightblue.spec [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+BuildRequires:  python3-devel
+BuildRequires:  python2-setuptools
+BuildRequires:  python3-setuptools
+BuildRequires:  python2-nose
+BuildRequires:  python3-nose
+
+BuildArch:      noarch
+
+%description
+A Python API for Lightblue database.
```
Comment (sochotnicky) [inline python-lightblue.spec]: Add a link to lightlbue.io so it's clear what is Lightblue :-)

**Comment by sochotnicky** [comment]
Maybe let's just close this off? I don't think it's strictly necessary?

**Comment by Allda** [comment]
Yep, specfile is already in Dist-git.

**Pull request closed without merging**